### PR TITLE
assemblystats 1.1.0

### PR DESCRIPTION
**Pull request by hellothisisMatt**
* #190 Rewrote Python script for Python 3 compatibility
* #95 Renamed README to README.rst
* #94 Bumped version to 1.1.0
* #93 Returns error code from .pl file, and exit reason
* #191 Update to gnuplot 5.2.7

### CODE REVIEW

File: tools/assemblystats/assembly_stats_txt.xml [hunk 1]
```diff
@@ -1,31 +1,34 @@
-<tool id="assemblystats" name="assemblystats" version="1.0.1">
+<tool id="assemblystats" name="assemblystats" version="1.1.0">
 	<description>Summarise an assembly (e.g. N50 metrics)</description>
 	<requirements>
 		<requirement type="package" version="1.6.924">perl-bioperl</requirement>
-		<requirement type="package" version="5.0.4">gnuplot</requirement>
+		<requirement type="package" version="5.2.7">gnuplot</requirement>
```
Comment (ericenns) [inline tools/assemblystats/assembly_stats_txt.xml:5]: Should also specify python 3 here as a dependency

Comment (hellothisisMatt) [reply tools/assemblystats/assembly_stats_txt.xml:5]: Good idea, added!

**Comment by hellothisisMatt** [comment]
Thanks to @DarianHole for fixing the conda issue 🏆

**Comment by DarianHole** [comment]
idk, I think it just installs correctly 1/10 tries 📦

**Pull request merged**